api/api_controller_test.go

```diff
@@ -1311,6 +1312,31 @@ func TestHandler_RetentionPolicyHandlers(t *testing.T) {
 	})
 }
 
+func TestHandler_ConfigHandlers(t *testing.T) {
+	handler, deps := getHandler(t)
```

Comment:
Nice test!

You rely on the blockstore type to be "mem", which is the default. It's ok that we're using mem (in-memory) type for the test, but it would be better to specify it explicitly, rather than relying on it as a default.

I suggest adding a `blockstoreType` arg to `getHandler` (before the opts).
If this arg is specified, the handler will be created with the given blockstoreType.

Currently, this value is taken from an environment variable (see NewBlockAdapterByEnv in handler_test.go) - keep this as the fallback option if the new argument is empty.